wasmbindings/test_utils.go

```diff
@@ -0,0 +1,48 @@
+package wasmbinding
+
+import (
+	"crypto/rsa"
+	"os"
+
+	"github.com/golang-jwt/jwt/v5"
```

Comment:
Please rewrite this with lestrrat-go, you can use the query handler as an example. We don't want to have two distinct jwk/t libraries in use